### ISSUE TITLE
[Feature Request] Hide Playlist Labels

### ISSUE THREAD

**Issue by Satanarious** [duffey/spicetify-playlist-labels#31]
There's already a button on the playbar that toggles showing/hiding 'All Saved Playlists'. I would like something to toggle showing/hiding playlist labels. Maybe you can configure the button I mentioned earlier to have 3 functions, which are cycled:
- Show All Playlist Labels
- Show My Playlist Labels
- Hide Playlist Labels

**Comment by stefb02** [comment]
what button are you talking about? I dont see it...

**Comment by Satanarious** [comment]
The spotify icon on the playbar in the bottom right

**Comment by zachpoblete** [comment]
Yes, I would really like to see this feature added because I have so many playlists that it clutters up the labels.

**Comment by zachpoblete** [comment]
@Satanarious Btw, I don't understand what the "Show All Saved Playlists" button does.

**Comment by Satanarious** [comment]
> @Satanarious Btw, I don't understand what the "Show All Saved Playlists" button does.

Right now it switches between playlists made by you and all playlists in your library(including the ones that are not created by you).

I want the switch to work in such a way that, on first click it shows all playlists in my library(including the ones that are not created by me). On second click, it should show all playlists that are created by me and on next click it should completely hide the playlist labels column.

I want this to be repeated as you click the button. Or maybe create a menu for the button to select between the three options.

**Comment by zachpoblete** [comment]
Ah, I see.

>I want the switch to work in such a way that, on first click it shows all playlists in my library(including the ones that are not created by me). On second click, it should show all playlists that are created by me and on next click it should completely hide the playlist labels column.

Yup, that sounds reasonable. I suggest that there should be a notification/tooltip when the label setting changes or a change to the appearance of the button to indicate the current label setting.

**Comment by lichenophile** [comment]
A toggle would be immensely helpful, this extension chugs spotify on larger playlists
